stdlib/tempfile.pyi

```diff
@@ -151,7 +151,7 @@ else:
             suffix: Optional[AnyStr] = ...,
             prefix: Optional[AnyStr] = ...,
             dir: Optional[_DirT[AnyStr]] = ...,
-        ) -> IO[bytes]: ...
+        ) -> _TemporaryFileWrapper[bytes]: ...
```

Comment:
Are you sure about this one?

```
Python 3.7.3 (default, Jan 22 2021, 20:04:44) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tempfile
>>> tempfile.TemporaryFile()
<_io.BufferedRandom name=3>
```

Reply:
I believe that is a mistake, yes. I was fighting with git and my dyslexia pretty hard and probably transposed the changes between the two classes by accident. The changed return type should only be on NamedTemporaryFile based on my reading of the source code. Will fix it when I work on this next. Thanks for noticing.

Reply:
I believe I've fixed it now.